examples/guestbook/README.md

```diff
@@ -46,30 +54,35 @@ Use the file `examples/guestbook/redis-master.json` which describes a single pod
 Create the redis pod in your Kubernetes cluster by running:
 
 ```shell
-$ cluster/kubectl.sh create -f examples/guestbook/redis-master.json
+$ cluster/kubectl.sh create -f examples/guestbook/redis-master-controller.json
+
+$ cluster/kubectl.sh get rc
+CONTROLLER                             CONTAINER(S)            IMAGE(S)                                 SELECTOR                     REPLICAS
+redis-master-controller                redis-master            dockerfile/redis                         name=redis-master            1
 ```
 
 Once that's up you can list the pods in the cluster, to verify that the master is running:
 
 ```shell
-cluster/kubectl.sh get pods
+$ cluster/kubectl.sh get pods
 ```
 
 You'll see a single redis master pod. It will also display the machine that the pod is running on once it gets placed (may take up to thirty seconds):
 
 ```
-NAME                IMAGE(S)            HOST                                                       LABELS              STATUS
-redis-master        dockerfile/redis    kubernetes-minion-2.c.myproject.internal/130.211.156.189   name=redis-master   Running
+POD                                          IP                  CONTAINER(S)            IMAGE(S)                                 HOST                                                           LABELS                                                     STATUS                                                                                                                               
+redis-master-controller-gb50a                10.244.3.7          redis-master            dockerfile/redis                         kubernetes-minion-4.c.hazel-mote-834.internal/104.154.54.203   app=redis,name=redis-master                                Running
```

Comment:
It's actually because the minions are now created as part of Managed Instance Groups members of which just have unique IDs versus counts.